db/articles.js

```diff
@@ -0,0 +1,111 @@
+'use strict';
+
+let articles = [];
+
+function post(propertyParams){
```

Comment:
Keep the naming of your methods consistent. So if you're using getArticle, you might as well name this createArticle.